tools/cli/cliRouter.js

```diff
@@ -35,10 +35,10 @@ export async function cli() {
 	argv = watchDefine( argv );
 
 	// This adds usage information on failure and demands that a subcommand must be passed.
-	argv.showHelpOnFail( true ).demandCommand();
+	argv.showHelpOnFail( true ).demandCommand().version( false );
```

Comment:
The removes the help text offering `--version` as a valid flag and stops it from returning. Since this is unpublished, there's no version incrementation occurring.